full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -128,6 +129,135 @@ describe("CloudSqlite", () => {
+      containerId,
+      accessToken: sasToken,
+      cloudSqliteLogId: "ContainerIdentifier",
+      writeable: true,
+    };
+    const container = CloudSqlite.createCloudContainer(containerProps);
+    container.initializeContainer();
+    container.connect(caches[1]);
+
+    let rows = container.queryHttpLog();
+    expect(rows.length).to.equal(2); // manifest and bcv_kv GETs.
+
+    // endTime to exclude these first 2 entries in later queries.
+    await BeDuration.wait(10);
+    const endTime = new Date().toISOString();
+
+    rows = container.queryHttpLog({startFromId: 2});
+    expect(rows.length).to.equal(1);
+    expect(rows[0].id).to.equal(2);
+
+    container.acquireWriteLock("test");
+    await CloudSqlite.uploadDb(container, {localFileName: testBimFileName, dbName: testBimFileName});
+    container.releaseWriteLock();
+
+    // 6 entries added by grabbing the write lock and checking for changes.
+    // 2 entries from before. Expect 6 total entries because we're filtering by endTime from before.
+    rows = container.queryHttpLog({finishedAtOrAfterTime: endTime, startFromId: 1});
+    expect(rows.length).to.equal(6);
+    expect(rows.find((value) => {
+      return value.id === 1 || value.id === 2;
+    })).to.equal(undefined);
+
+    rows = container.queryHttpLog({finishedAtOrAfterTime: endTime});
+    expect(rows.length).to.equal(6);
+    expect(rows.find((value) => {
+      return value.id === 1 || value.id === 2;
+    })).to.equal(undefined);
+
+    rows = container.queryHttpLog({finishedAtOrAfterTime: endTime, startFromId: 1, showOnlyFinished: true});
+    expect(rows.length).to.equal(6);
+    expect(rows.find((value) => {
+      return value.id === 1 || value.id === 2;
+    })).to.equal(undefined);
+
+    rows = container.queryHttpLog({showOnlyFinished: true});
+    expect(rows.length).to.equal(8);
+
+    rows = container.queryHttpLog({startFromId: 4, showOnlyFinished: true});
+    expect(rows.length).to.equal(5);
+
+  });
+
+  it("should pass cloudSqliteLogId through container to database", async () => {
+    const containerId = Guid.createValue();
+    const sasToken = CloudSqliteTest.makeSasToken(containerId,"racwdl");
+    const containerProps: CloudSqlite.ContainerAccessProps = {
+      accessName: CloudSqliteTest.storage.accessName,
+      storageType: CloudSqliteTest.storage.storageType,
+      containerId,
+      accessToken: sasToken,
+      cloudSqliteLogId: "ContainerIdentifier",
+      writeable: true,
+    };
+    const container = CloudSqlite.createCloudContainer(containerProps);
+    container.initializeContainer();
+    container.connect(caches[1]);
+
+    container.acquireWriteLock("test");
+    await CloudSqlite.uploadDb(container, {localFileName: testBimFileName, dbName: basename(testBimFileName)});
+    container.releaseWriteLock();
+    container.checkForChanges();
+
+    let db: IModelJsNative.DgnDb = IModelDb.openDgnDb({ path: basename(testBimFileName) }, OpenMode.Readonly, undefined, {container});
```

Comment:
remove type override

Reply:
Why is that a type override? Thats just the type that IModelDb.openDgnDb returns, right?

Reply:
yes, the type is defined by the method, you should not override it.

Reply:
Is it a bad thing to explicitly mention what type I'm going to get? I don't think I can override in this way, I would get a build error if I put let db: number

Reply:
yes it's a bad thing.